<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QMainWindow" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ObjTreeWgt" name="objsList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>World</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>979</x>
      <y>0</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>&amp;Create</string>
    </property>
    <addaction name="actionScene"/>
   </widget>
   <widget class="QMenu" name="menuObjects">
    <property name="title">
     <string>Ob&amp;jects</string>
    </property>
    <addaction name="actionNew_Object"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCreate"/>
   <addaction name="menuObjects"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionScene">
   <property name="text">
    <string>&amp;Scene</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Sa&amp;ve As</string>
   </property>
  </action>
  <action name="actionNew_Object">
   <property name="text">
    <string>&amp;New Object</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjTreeWgt</class>
   <extends>QTreeWidget</extends>
   <header>ProjEd/headers/ProjectEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
